src/meta/src/rpc/elections.rs

```diff
@@ -393,6 +395,13 @@ pub async fn run_elections<S: MetaStore>(
 
                     if let Ok(leader_alive) =
                         manage_term(is_leader, &leader_info, lease_time_sec, &meta_store).await && !leader_alive {
+                            // Leader lost leadership. Trigger fencing
+                            if is_leader {
+                                leader_tx.send((MetaLeaderInfo{
+                                    node_address: "".to_owned(), 
+                                    lease_id: 0
```

Comment:
`MetaLeaderInfo` implements Default, you can directly use `::default()`

Reply:
Or can we just panic here at this stage?

Reply:
I will exit directly. I moved the fencing logic to `elections.rs` (see [ece527c](https://github.com/risingwavelabs/risingwave/pull/7022/commits/ece527c0e108aa06c21b01d4416e3736b4c0e2c1))